.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+((nil
+  .
+  ((cider-clojure-cli-aliases
+    .
+    "dev"))))
```

Comment:
@daslu  What is this doing?

Reply:
Making sure the `:dev` alias is used from `deps.edn`, so we have the Clay dependency in our dev environment.